quantum/action_tapping.c

```diff
@@ -362,7 +362,7 @@ bool process_tapping(keyrecord_t *keyp) {
 #        ifdef RETRO_TAPPING_PER_KEY
                 get_retro_tapping(tapping_keycode, keyp) &&
 #        endif
-                (RETRO_SHIFT + 0) != 0 && TIMER_DIFF_16(event.time, tapping_key.event.time) < (RETRO_SHIFT + 0)
+                ((RETRO_SHIFT + 0) == 0 || TIMER_DIFF_16(event.time, tapping_key.event.time) < (RETRO_SHIFT + 0))
```

Comment:
```suggestion
                // if RETRO_SHIFT is not defined with an explicit value (`#define RETRO_SHIFT`) OR
                // the time that separates this key event from the tapping key event is below the retro shift term defined by the user (`#define RETRO_SHIFT <value>`)
                ((RETRO_SHIFT + 0) == 0 || TIMER_DIFF_16(event.time, tapping_key.event.time) < (RETRO_SHIFT + 0))
```
A second comment saying the same thing on the same line of code might not be very helpful but in my opinion, it doesn't hurt to repeat it.